local/build-binary

```diff
@@ -235,4 +237,17 @@ cp -r ${WORKDIR}/unittest/gunit/*  ${INSTALL_DIR}/usr/local/${PRODUCT_FULL}/
 cp -r ${WORKDIR}/runtime_output_directory ${INSTALL_DIR}/usr/local/${PRODUCT_FULL}/
 cp -r ${WORKDIR}/library_output_directory ${INSTALL_DIR}/usr/local/${PRODUCT_FULL}/
 cp -r ${WORKDIR}/plugin_output_directory ${INSTALL_DIR}/usr/local/${PRODUCT_FULL}/
+
+# GCOV instrumentation files for coverage report. 
+if [[ ${WITH_GCOV} == 'ON' ]]; then
+    pushd ${SOURCEDIR}
+    wget https://raw.githubusercontent.com/Percona-QA/percona-qa/master/Coverage/dgcov.pl
+    perl dgcov.pl --v --dump --source=$SOURCEDIR
```

Comment:
We can improve `dgcov.pl` so that it would take `--dump=<path_to_dump_file>` to avoid the `cp` on the next line.

Reply:
thanks good suggestion will add incrementally.